### PR TITLE
Updated PackageValidationBaselineVersion

### PR DESCRIPTION
**Pull request by MichalPavlik**
Fixes partially #10248

Part of release process

### CODE REVIEW

**Review by JanKrivanek — APPROVED** [review]
This is fine - it just needs to wait for 17.11 packages to get pushed to nuget feed after GA (https://dev.azure.com/devdiv/DevDiv/_wiki/wikis/DevDiv.wiki/39798/Dev17.11-Schedule)

File: src/Tasks/CompatibilitySuppressions.xml [hunk 1]
```diff
@@ -1,10 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <!-- https://learn.microsoft.com/en-us/dotnet/fundamentals/package-validation/diagnostic-ids -->
 <Suppressions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
-<!-- For ease of logging the "not supported on Core" message, these tasks are a
-         TaskRequiresFramework on netstandard/netcore. Since the type is sealed there,
-         that shouldn't cause any implementation problems since no one can derive
-         from it and try to call TaskExtension.Log. -->
   <Suppression>
```
Comment (rainersigwald) [inline src/Tasks/CompatibilitySuppressions.xml:4]: Please keep these comments (it might be easiest to just revert the changes to these files).

**Review by rainersigwald — CHANGES_REQUESTED** [review]

File: src/Tasks/CompatibilitySuppressions.xml (hunk 1 above)
Comment (MichalPavlik) [reply src/Tasks/CompatibilitySuppressions.xml]: It was easy to just copy them back. There is one alignment change, but they are back :)

**Review by rainersigwald — APPROVED** [review]
